mac/deploy_mac.sh

```diff
@@ -67,18 +67,23 @@ build_app()
 
 build_installer_image()
 {
-    # Install dmgbuild (for the current user), this is required to build the installer image
-    python -m ensurepip --user --default-pip
-    python -m pip install --user dmgbuild==1.4.2
-    local dmgbuild_bin="$(python -c 'import site; print(site.USER_BASE)')/bin/dmgbuild"
-
+    # this needs brew to be installed first
+    # install create-dmg
+    brew install create-dmg
     # Get Jamulus version
     local app_version="$(cat "${project_path}" | sed -nE 's/^VERSION *= *(.*)$/\1/p')"
-
+    # Copy the styling data (create-dmg doesn't allow creating .DS_Store files since it uses
+    # an apple script which can only run in GUI mode https://github.com/create-dmg/create-dmg/issues/72#issuecomment-564926248)
+    cp ${macdeploy_path}/DS_StoreInstaller ${deploy_path}/.DS_Store
```

Comment:
What is this?  16KB binary file?

Reply:
It's the .DS_store file from the old installer. Unfortunately it gets overwritten. I don't really think this create-dmg script is the correct tool for us (much too hacky). Maybe we should use a node-js version instead: https://github.com/sindresorhus/create-dmg

Reply:
Or this one: https://github.com/LinusU/node-appdmg

Reply:
Just found https://github.com/actions/virtual-environments/issues/553

Reply:
Urgh. Certainly overthought the whole stuff. It seems to work with GH actions!!!!